certify-service/src/main/java/io/mosip/certify/services/VCIssuanceServiceImpl.java

```diff
@@ -159,44 +153,76 @@ private VCResult<?> getVerifiableCredential(CredentialRequest credentialRequest,
         throw new CertifyException(ErrorConstants.VC_ISSUANCE_FAILED);
     }
 
-    private CredentialResponse<?> getCredentialResponse(String format, VCResult<?> vcResult) {
+    private CredentialResponse<?> getCredentialResponse(String format, VCResult<?> vcResult, String version) {
         switch (format) {
             case "ldp_vc":
                 CredentialResponse<JsonLDObject> ldpVcResponse = new CredentialResponse<>();
                 ldpVcResponse.setCredential((JsonLDObject)vcResult.getCredential());
-                ldpVcResponse.setFormat(vcResult.getFormat());
+                if (!isV13OrAbove(version)) {
+                    ldpVcResponse.setFormat(vcResult.getFormat());
+                }
                 return ldpVcResponse;
 
             case "jwt_vc_json-ld":
             case "jwt_vc_json":
                 CredentialResponse<String> jsonResponse = new CredentialResponse<>();
                 jsonResponse.setCredential((String)vcResult.getCredential());
-                jsonResponse.setFormat(vcResult.getFormat());
+                if (!isV13OrAbove(version)) {
+                    jsonResponse.setFormat(vcResult.getFormat());
+                }
                 return jsonResponse;
         }
         throw new CertifyException(ErrorConstants.UNSUPPORTED_VC_FORMAT);
     }
 
-    private Optional<CredentialMetadata>  getScopeCredentialMapping(String scope) {
-        LinkedHashMap<String, Object> vciMetadata = issuerMetadata.get("latest");
-        if(supportedCredentials == null) {
-            supportedCredentials = (LinkedHashMap<String, Object>) vciMetadata.get("credentials_supported");
+    protected Optional<CredentialMetadata>  getScopeCredentialMapping(String scope, String version) {
```

Comment:
So all changes to this function should be reverted? Reverting this will also lead to changing back the interface to not have a `version` parameter. Right now it has been added here

```java
public interface VCIssuanceService {

    /**
     *
     * @param credentialRequest
     * @return
     */
    <T> CredentialResponse<T> getCredential(CredentialRequest credentialRequest, String version);

```